### PR TITLE
RSDEV-280: Adding utility method into the MovableInventoryRecord object

### PR DESCRIPTION
**Pull request by rs-nicof**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
   <modelVersion>4.0.0</modelVersion>
   <artifactId>rspace-core-model</artifactId>
-  <version>2.0.1</version>
+  <version>2.0.2-NIK-SNAPSHOT</version>
```
Comment (mKowalski256) [inline pom.xml:7]: please use version based on issue number (i.e. 2.0.2-rsdev-280-SNAPSHOT)

File: src/main/java/com/researchspace/model/inventory/MovableInventoryRecord.java [hunk 2]
```diff
@@ -47,7 +45,17 @@ public Container getParentContainer() {
...
+	/**
+	 * @return true if the MovableInventoryRecord is currently
+	 * located into (linked to) a proper Container (so not in the workbench)
+	 */
+	@Transient
+	public boolean isLocatedIntoContainer() {
+		return this.getParentContainer() != null
+				&& !ContainerType.WORKBENCH.equals(
+				this.getParentContainer().getContainerType());
+	}
```
Comment (mKowalski256) [inline src/main/java/com/researchspace/model/inventory/MovableInventoryRecord.java:58]: looks good, but the naming seems a bit off, how about `isStoredInContainer`?

also Container class has `isWorkbench()` method, so the check could be simplified to 
```
this.getParentContainer() != null && !this.getParentContainer.isWorkbench()
```

**Review by mKowalski256 — APPROVED** [review]
Looks good, couple of minor comments.

File: src/main/java/com/researchspace/model/inventory/MovableInventoryRecord.java (hunk 2 above)
Comment (rs-nicof) [reply src/main/java/com/researchspace/model/inventory/MovableInventoryRecord.java]: changed

**Comment by rs-nicof** [comment]
Build passed: https://jenkins.researchspace.com/job/rspace-web-dependencies/job/rspace-core-model/job/RSDEV-280-core-model/5/

**Comment by mKowalski256** [comment]
The target of this PR should be rspace-os/rspace-core-model/main, rather than ResearchSpace-ELN/rspace-core-model/main - I'll close and recreate PR for os repo.

**Pull request closed without merging**